### PR TITLE
Add "send confirmation" option

### PR DESCRIPTION
**Pull request by jlerner**
Dangerous keyboard shortcut workaround.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -87,6 +87,19 @@ function FastMailEnhancementSuite(options) {
 
   // Features /////////////////////////////////////////////////////////////////
 
+  $.option('send_confirmation', function () {
```
Comment (lamby) [inline js/main.js:90]: I think you are missing the corresponding option on the option page, no?

**Review by lamby — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (jlerner) [reply js/main.js:90]: Sorry, it was inadvertently excluded from the commit.

File: pages/options.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
 var FastMailEnhancementSuiteOptions = [
     {
+      "name": "send_confirmation"
+    , "title": "Send confirmation"
+    , "description": "Requires confirmation to send email. Handy for users of FastMail keyboard shortcuts."
+    , "image": false
+    , "default": true
```
Comment (lamby) [inline pages/options.js]: Please change the default to `false`.

**Review by lamby — CHANGES_REQUESTED** [review]

**Comment by lamby** [comment]
Many thanks :)

**Comment by lamby** [comment]
Published on the app store as `1.4.0`

**Comment by jlerner** [comment]
Thanks for this extension in the first place!

**Pull request merged**